packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -333,6 +333,11 @@ class WebAssetServer implements AssetReader {
   // handle requests for JavaScript source, dart sources maps, or asset files.
   @visibleForTesting
   Future<shelf.Response> handleRequest(shelf.Request request) async {
+    if (request.method == 'POST') {
```

Comment:
Should this be negated to only respond for GET instead? not that PUT is particularly common

Reply:
Done.